src/components/BookList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const BookList = () => (
+  <div className="d-flex flex-row justify-content-center border">
+    <div className="d-flex flex-column px-3">
+      <h3>Programming</h3>
+      <h1>React for Dummies</h1>
+      <span>Author&apos;s name</span>
```

Comment:
* I strongly suggest you make this component take in and render book data ( `Title` and `Author`) as props, this is good practice as props enable you to pass properties from one component to another 
* For example you can render the `BookList` component multiple times with different data [Checkout this post on how to use Props](https://itnext.io/what-is-props-and-how-to-use-it-in-react-da307f500da0)

Reply:
Thank You So much for the review, but this is just the first milestone of the project and I used dummy data, I will literally have to change everything in my code anyways.